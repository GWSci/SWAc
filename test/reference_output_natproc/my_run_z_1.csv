DATE,nDays,Area,Precipitation,PEt,VegetationPE_PEfac,CanopyStorage,PEfacLessCanopy_AE,Precipitation_Groundlevel,Snowfall,Precipitation_Rainfall,SnowPack,SnowMelt,Rainfall_SnowMelt,RapidRunoff,RunoffRecharge,MacroPoreAttenuated,MacroPoreDirect,Percolation_RootZone,RAWREW,TAWTEW,pSMD,SMD,AE,RejectedRecharge,PercolationThroughRootZone,SubRootZoneLeakege,BypassingInterflow,InputInterflowStore,InterflowStoreVol,InfiltrationRecharge,InterflowtoSW,InputRechargeStore,RechargeStoreVol,CombinedRecharge,SWAttenInputPot,SWAttenuation,PondDirect,PondAtten,OpenWaterAE,SWAttenInputAct,PondOverspill,OtherSW,OpenWaterPE,SWAbstractions,SWDischarges,CombinedSW,CombinedAE,UnitilisedPE,AVERAGE_IN,AVERAGE_OUT,TOTAL_STORAGE_CHANGE,BALANCE
05/01/1980,5,999.4499999999999,7.479983745,18.860588709677437,7.544235483870976,0.06731985370500007,7.469435646420975,6.664665516795002,5.1487703871676525,2.256413520382347,38.3586557162106,104.92835920962273,107.18477273000507,15.871968300560766,1.5559081595447557,0.0,0.0,80.5943271564438,634.9006125,1154.36475,103.1249543576324,177.66903864047524,6.050242873600989,0.0,0.0,0.2498625,0.011243812500000004,0.21475681875000005,30.674375197036873,3.1482674999999998,1.3763053848518436,5.065226972044755,16.76474394472544,0.8382371972362717,2.2488447535171603,2.4324079234021507,0.0,0.0,1.5396958488930619,2.3273874159144223,1.7134848016223105,2.0424235947966265,1.618238511290324,0.0,499.94999999999993,470.3759083964189,7.657258576199053,0.6722492081778876,507.6798462449999,478.87140416985426,72.70791576104492,-43.89947368589918
30/09/2014,12687,999.4499999999999,42021.15159323025,42903.142590380194,38366.38940816823,374.0861047240826,37950.738180697015,37444.95032918314,21572.21358369512,20033.286782063926,2842800.86821019,21572.378994872663,41605.66577693659,7349.074288857448,604.2533051096611,212.42942479793714,141.61961653195812,29741.975869055586,1610996.8141575002,2929085.1166500007,1610833.4001701989,1626299.2117136738,14276.164325580712,11020.454644530037,4629.901661430231,634.0011074999999,260.02513290901163,4966.480038562121,55692.79298790182,2300.4576299075534,2669.6167678997135,3632.771896047225,72737.76823059906,3778.508028061911,22529.776342191897,5566.866084039042,0.0,0.0,1892.2987154234397,22744.70884007712,20060.835933124905,2682.669854254038,2107.2312133086643,0.0,2748625.11,2771368.6157873794,16542.549145728237,19879.500037046604,2791280.2627007305,2791689.6729611694,-383.25887708815117,-26.151383350734346
