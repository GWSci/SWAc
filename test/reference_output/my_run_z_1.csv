DATE,nDays,Area,Precipitation,PEt,VegetationPE_PEfac,CanopyStorage,PEfacLessCanopy_AE,Precipitation_Groundlevel,Snowfall,Precipitation_Rainfall,SnowPack,SnowMelt,Rainfall_SnowMelt,RapidRunoff,RunoffRecharge,MacroPoreAttenuated,MacroPoreDirect,Percolation_RootZone,RAWREW,TAWTEW,pSMD,SMD,AE,RejectedRecharge,PercolationThroughRootZone,SubRootZoneLeakege,BypassingInterflow,InputInterflowStore,InterflowStoreVol,InfiltrationRecharge,InterflowtoSW,InputRechargeStore,RechargeStoreVol,CombinedRecharge,SWAttenuation,SWAbstractions,SWDischarges,CombinedSW,CombinedAE,UnitilisedPE,AVERAGE_IN,AVERAGE_OUT,TOTAL_STORAGE_CHANGE,BALANCE
01/01/1980,1,999.4499999999999,0.0,3.7721177419354874,1.5088470967741952,0.0,1.5088470967741952,0.0,0.0,0.0,24.646160945615,75.298839054385,75.298839054385,15.059767810877,0.0,0.0,0.0,60.23907124350799,126.98012249999998,230.87294999999997,41.21477585326621,99.94500000000001,1.5088470967741952,0.0,0.0,0.049972499999999996,0.0024986250000000004,0.047473875,8.830140750000002,0.6996149999999999,0.4065262875000001,0.702113625,0.99945,0.049972499999999996,14.938723229513098,0.0,33.32999999999999,34.8570208688639,1.5088470967741952,0.0,33.379972499999994,36.41584046563809,58.32369785923379,-61.35956582487189
30/09/2014,12691,999.4499999999999,42028.63157697525,42918.23106134794,38372.42479655532,415.7260273086529,37956.69876924667,41612.9055496666,21577.362354082292,20035.543195584305,2842814.5807049605,21602.008515027897,41637.551710612206,8089.0462092294365,0.0,230.9318504764026,153.9545669842684,33163.6190839221,1611504.7346475,2930008.60845,1846700.5365328963,1863282.1947879854,16581.960828832962,11890.852129344821,4873.47146320621,634.2009974999999,275.38362303531045,5232.288837670899,57757.96163794698,2475.9102686695223,2764.102568463873,2982.225742181235,59655.61186147404,3136.7351600579705,1187889.643905766,0.0,2749091.73,2771742.502446787,16997.686856141612,21374.737940413706,2791754.5625744755,2791876.924462986,-97.71572756569611,-24.64616094564289
